README.md

```diff
@@ -86,7 +86,6 @@ That section is intended for developers who wish to build pypowsybl from the sou
 
 Requirements:
 
-- Maven >= 3.1
```

Comment:
should it be maven>=3.8 ?

Reply:
mvnw is managing by itself ?

Reply:
Yes, so as it is fully automatic developper do not need anymore to be aware on maven version